### PR TITLE
Using master_name from config if set.

### PR DESCRIPTION
**Pull request by mrunesson**
We needed to set a master_name explicitly  in the case of long hostnames and the base64 addition to active_tokens_ table (and others) makes the table too long (more than 64 chars that MySQL and Postgres restrict on). 
We then noticed when running:
python -m pinball.tools.workflow_util -c path/to/pinball/yaml/configuration/file -f reschedue
that the workflow_util does not set the master_name override from the config.

### CODE REVIEW

File: pinball/tools/workflow_util.py [hunk 1]
```diff
@@ -45,6 +45,7 @@
 from pinball.workflow.utils import get_logs_dir
 from pinball.workflow.utils import get_unique_workflow_instance
 from pinball.workflow.utils import load_path
+from pinball.config.utils import master_name
```
Comment (MaoYe) [inline pinball/tools/workflow_util.py]: please order the import statement in alphabetical order.

**Comment by MaoYe** [comment]
Other the the minor comment, it looks good to me. Thanks!

**Comment by ygoldman** [comment]
@MaoYe will this PR be merged in? I can open a new one with the minor comments addressed - ran into a similar issue...

**Comment by MaoYe** [comment]
@ygoldman please create a new one, and address the comments. Then we can close this. Thanks a lot!

**Comment by ygoldman** [comment]
Done. https://github.com/pinterest/pinball/pull/36. And you're very welcome! We are exploring your tool.  Great work and thanks a lot for open sourcing!

**Comment by MaoYe** [comment]
close this as it is solved in another thread
https://github.com/pinterest/pinball/pull/36

**Pull request closed without merging**